build/Data/CHARS/Sonic/Sonic.txt

```diff
@@ -318,6 +323,8 @@ anim	freespecial2
 	frame	data/chars/Sonic/318.gif
 		
 anim	freespecial3
+	@cmd	changeentityproperty getlocalvar("self") "mp" 0
```

Comment:
One note @KoenPreston  the command to use MP would go under the isavailable command. Because if they don't have the right power level, you don't want MP to get used up.